master/internal/api_agents.go

```diff
@@ -34,6 +34,7 @@ func (a *apiServer) GetAgents(
 }
 
 func (a *apiServer) GetAgent(
+	// TODO: Does this work on k8s?
```

Comment:
@aaron276h I'm a little confused here. In `GetAgents`, we check if we are using k8s and then either route the request to `/agents` or to `/pods` accordingly. We don't seem to be doing the same thing here. What is supposed to happen with this API on k8s?

Reply:
Great question. As is it won't work since we don't really have a concept of an agent in Kubernetes anyways. We could add in logic to route this request to `pods` actor and have it handle it by translating to a k8s node, not sure if the value add is enough to justify the work though.

Reply:
That makes sense. I'm going to leave it as is then.